models/engine/db_storage.py

```diff
@@ -74,3 +74,17 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Retrieve one object"""
```

Comment:
You may need to add "self.__session.query(models.classes[cls]).all()" to the db storage functions as we are pulling from the SQL database.